tests/parser/StringTests.cpp

```diff
@@ -221,3 +221,19 @@ BOOST_AUTO_TEST_CASE(replace_all) {
     replaceAll<std::string>(s1, "m", "foo");
     BOOST_CHECK_EQUAL(s1, "lorefoo ipsufoo");
 }
+
+
+BOOST_AUTO_TEST_CASE(split) {
+    std::string s1 = "lorem ipsum";
+
+    auto split1 = split_string(s1, ' ');
+    BOOST_CHECK_EQUAL(split1.size(), 2);
+    BOOST_CHECK_EQUAL(split1[0], "lorem");
+    BOOST_CHECK_EQUAL(split1[1], "ipsum");
+
+    auto split2 = split_string(s1, "r ");
```

Comment:
How about `split_string(s1, "m ")`

Reply:
it would not work well. fixed, thanks for suggestion.